tests/Unit/Domain/Test_Block.cpp

```diff
@@ -351,5 +357,13 @@ SPECTRE_TEST_CASE("Unit.Domain.Block", "[Domain][Unit]") {
     const Block<2> rhs(identity_map.get_clone(), 0, neighbors, "Identity");
     CHECK(block != rhs);
   }
+  {
+    const Block<2> rhs(
+        identity_map.get_clone(), 3, neighbors, "Identity",
+        {{Spectral::Basis::Chebyshev, Spectral::Basis::Legendre}});
+    CHECK(rhs.dg_basis() ==
+          std::array{Spectral::Basis::Chebyshev, Spectral::Basis::Legendre});
+    CHECK(block != rhs);
+  }
```

Comment:
Could you add a test for Ylm basis? Mostly just to provide an example for folks in the future with a 2d basis.

Reply:
Such a test does not make sense until the next PR which fixes assumptions about external boundaries.   So I can either add the commits for that PR to this one, or you can wait, doesn't matter to me...